lib/default-config.js

```diff
@@ -1100,21 +1100,20 @@ export default function (config) {
       // Array of choice arrays should be flattened.
       choices = _.compact(_.flatten(choices));
 
-      choices.forEach(function (choice, i) {
+      const choicesWithLabels = choices.map(function (choice) {
         // Convert any string choices to objects with `value` and `label`
```

Comment:
Previously, iterating over these and mutating the original `choices` array would throw an error if passed an immutable object. This resolves that by not mutating the items in the `choices` array, but providing a new array with modifications, if necessary. If all the choices are not a string and have a label, this should return a copy of the original array.

Reply:
Bah, mutation bad, functional good. 👍 !